### ISSUE TITLE
Chart data not rendering properly when "unloadBeforeLoad" prop is used 

### ISSUE THREAD

**Issue by miguelCT** [planttheidea/react-billboardjs#2]
Hi,

I've been trying to use **unloadBeforeLoad** prop to clean the chart data before loading new data and I think there's a problem.

The exact problem is that not every data is unloaded and I think it's because of this warning in **billboardjs** API: 
> **Unload**: Note: If you call load API soon after/before unload, unload param of load should be used. Otherwise chart will not be rendered properly because of cancel of animation.
[Source doc](https://naver.github.io/billboard.js/release/latest/doc/Chart.html#unload)

The exact point in react-billboardjs code is this code:

```js
if (unloadBeforeLoad) {
  instance.unloadData();
}

instance.loadData(data);
```

should be changed to

```js
instance.loadData({...data, unload: unloadBeforeLoad});
```
 https://github.com/planttheidea/react-billboardjs/blob/master/src/index.js#L209

I have fixed it in my local fork and it works perfect. 
I can submit a PR for you to review it and decide if you want to include the changes.

Cheers.

**Comment by planttheidea** [comment]
Good find! I left a comment on the PR, once addressed we should get this loaded ASAP. Appreciate the contribution!

**Comment by miguelCT** [comment]
Hi, I fixed it a couple of days ago.

I was wondering if you're planning in including these changes and publishing a new version in the next days. I don't want to put pressure on you. If you are going to wait, I can publish an internal version in my company's private registry in the meanwhile.

**Comment by planttheidea** [comment]
No no all good, for some reason I didn't get notified of the update to the PR. I'll try to publish a new version tonight when I get home.

**Comment by miguelCT** [comment]
Perfect. Thanks!

**Comment by planttheidea** [comment]
Published as part of `1.2.0`.
